jumpscripts/collectors_reality/network_gathering.py

```diff
@@ -20,9 +20,25 @@
 
 
 def action():
+
+    startswith_filter = ("vx-", "space_", "gw_", "gwm-", "spc-")
     ncl = j.clients.osis.getCategory(j.core.osis.client, "system", "nic")
+    nodecl = j.clients.osis.getCategory(j.core.osis.client, "system", "node")
+    netinfo = j.system.net.getNetworkInfo(startwith_filter=startswith_filter)
+    node = nodecl.get(j.application.whoAmI.nid)
+    change = False
+    node_addr = j.system.net._networkInfoFilter(node.netaddr, startswith_filter)
+    for nodenet in node_addr:
+        for net in netinfo:
+            if nodenet["name"] == net["name"]:
+                if nodenet != net:
+                    change = True
+                    break
+
+    if change:
+        node.netaddr = netinfo
+        nodecl.set(node)
```

Comment:
can we do this with an updateSearch instead? To prevent other attribute changes mess?